integration/test/performance_tracker/performance_metrics_test.go

```diff
@@ -0,0 +1,46 @@
+//go:build linux && integration
+// +build linux,integration
+
+package data_collector
+
+import(
+	"testing"
+	"time"
+	"log"
+
+	"github.com/aws/amazon-cloudwatch-agent/integration/test"
+)
+
+const (
+	configPath = "resources/config.json"
+	configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
+	agentRuntime = 20 //minutes
+)
+
+func PerformanceTest(t *testing.T) {
+
+	instanceId := test.GetInstanceId()
+	log.Printf("Instance ID used for performance metrics : %s\n", instanceId)
+
+	test.CopyFile(configPath, configOutputPath)
+
+	test.StartAgent(configOutputPath, true)
+
+	//let agent run before collecting performance metrics on it
+	time.Sleep(agentRuntime * time.Minute)
+	log.Printf("Agent has been running for : %s\n", (agentRuntime * time.Minute).String())
```

Comment:
We can define the `time.Duration` value once rather than calculating it twice
```suggestion
        agentRuntimeMin := agentRuntime * time.Minute
	time.Sleep(agentRuntimeMin)
	log.Printf("Agent has been running for : %s\n", agentRuntimeMin.String())
```